File: cordapp-cpk2/src/test/resources/multi-version-cordapp/src/main/java11/com/example/multi/ExampleContract.java
package com.example.multi;

import net.corda.v5.ledger.contracts.Contract;
import net.corda.v5.ledger.transactions.LedgerTransaction;
import org.slf4j.LoggerFactory;
import org.slf4j.Logger;

public class ExampleContract implements Contract {
    private final Logger LOGGER = LoggerFactory.getLogger(ExampleContract.class);

    @Override
    public void verify(LedgerTransaction ltx) {
        LOGGER.info("Java 11: {}", ltx);
    }
}

File: cordapp-cpk/src/test/resources/cordapp-with-platform/src/main/kotlin/com/example/cpk/platform/ExampleContract.kt
@file:Suppress("PackageDirectoryMismatch")
package com.example.cpk.platform

import net.corda.v5.ledger.contracts.Contract
import net.corda.v5.ledger.transactions.LedgerTransaction
import org.slf4j.Logger
import org.slf4j.LoggerFactory

class ExampleContract : Contract {
    private val logger: Logger = LoggerFactory.getLogger(ExampleContract::class.java)

    override fun verify(ltx: LedgerTransaction) {
        logger.info("LTX: {}", ltx)
    }
}
